### ISSUE TITLE
Training Loss NaN

### ISSUE THREAD

**Issue by HencyChen** [wdkyh/DS-MVSNet#2]
Hi @wdkyh ,

Thanks for the great work!

I'm trying to execute the training code by setting the proper dataset & output path. Though the code can start training smoothly, however, the loss starts to increase and reach NaN in the second iteration as following:

![image](https://user-images.githubusercontent.com/17294835/204293285-245afe88-dec3-4eb0-aeb1-7d9b15fa5c6f.png)

I'm wondering if there's any suggestion or other command argument example to reproduce the paper results?

Thanks again!

**Comment by wdkyh** [comment]
Hi @HencyChen,

Did you use the DTU dataset?

**Comment by HencyChen** [comment]
Hi @wdkyh,

Thanks for the reply. 

Yes I use DTU dataset

**Comment by HencyChen** [comment]
@wdkyh,

As for the distributed training, I follow the paper to set batch size=8 and also run on RTX 3090. However, I keep facing out-of-memory issue, even I set batch size to only 2. 

I'm wondering if there's any incorrect setting for distributed training?

Here's my training command:
`python -m torch.distributed.launch --nproc_per_node=4 main.py --agg_mode --batch_size 8`

**Comment by wdkyh** [comment]
Hi,
The batch size in each device is equal to 2, with 24G memory.

**Comment by HencyChen** [comment]
Hi @wdkyh,

So you mean the `--batch_size` argument should set to 2 rather than 8?

**Comment by HencyChen** [comment]
Hi @wdkyh,

After using distributed training with batch_size set to 2, the loss is still oscillating.
